packages/table/src/headers/columnHeader.tsx

```diff
@@ -78,13 +91,35 @@ export interface IColumnHeaderProps extends ISelectableProps, IColumnIndices, IC
     onResizeGuide: (guides: number[]) => void;
 }
 
+export interface IColumnHeaderState {
+    /**
+     * Whether the drag-select interaction has finished (via mouseup). When
+     * true, DragReorderable will know that it can override the click-and-drag
+     * interactions that would normally be reserved for drag-select behavior.
+     */
+    hasSelectionEnded?: boolean;
+}
+
 @PureRender
-export class ColumnHeader extends React.Component<IColumnHeaderProps, {}> {
+export class ColumnHeader extends React.Component<IColumnHeaderProps, IColumnHeaderState> {
     public static defaultProps = {
+        isReorderable: false,
         isResizable: true,
         loading: false,
     };
 
+    public state = {
+        hasSelectionEnded: false,
+    } as IColumnHeaderState;
```

Comment:
again, do it up front. and on one line. 
`public state: IColumnHeaderState = { hasSelectionEnded: false };`